packages/angular/src/flyout-menu/flyout-menu.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, Input, OnInit, ViewEncapsulation } from '@angular/core';
+import { IconService } from 'carbon-components-angular';
+import { Filter16 } from '@carbon/icons';
+
+/**
+ * [See demo](../../?path=/story/components-flyout-menu--basic)
+ *
+ * html:
+ * ```
+ * <ai-flyout-menu>
+ *	options
+ * </ai-flyout-menu>
+ * ```
+ *
+ * <example-url>../../iframe.html?id=components-flyout-menu--basic</example-url>
+ */
+@Component({
+  selector: 'ai-flyout-menu',
+  template: `
+    <ng-template #templateRef let-tooltip="tooltip">
+      <div class="bx--tooltip__content">
+        <div class="iot--flyout-menu--content">
+          <ng-content></ng-content>
+        </div>
+        <ng-content select="ai-flyout-menu-footer"></ng-content>
```

Comment:
Maybe also add `.iot--flyout-menu__bottom-container` so users can pass in a custom footer with the iot base class?
```suggestion
        <ng-content select="ai-flyout-menu-footer, .iot--flyout-menu__bottom-container"></ng-content>
```

It's tricky, since we don't have a content component, so we have to use selectors to put stuff in the right place ... but that should work for most cases?